containers/azure-terraform-0.12/README.md

```diff
@@ -0,0 +1,39 @@
+# Azure Terraform
+
+## Summary
+
+*Get going quickly with Terraform in Azure. Includes Terraform, the Azure CLI, the Docker CLI (for testing locally), Node.js for Cloud Shell, and related extensions and dependencies.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The VS Code Team |
```

Comment:
Definitely, take credit for your hard work here!  We appreciate it.

Reply:
Fixed in 30d13b9